compiler/damlc/daml-doc/src/DA/Daml/Doc/Annotate.hs

```diff
@@ -20,11 +20,18 @@ applyMove = map (foldr1 g) . groupSortOn (modulePriorityKey . md_name) . map f
             | Just new <- isMove md_descr = md{md_name = new, md_descr = Nothing}
             | otherwise = md
 
-        g m1 m2 = m1{md_adts = md_adts m1 ++ md_adts m2
-                    ,md_functions = md_functions m1 ++ md_functions m2
-                    ,md_templates = md_templates m2 ++ md_templates m2
-                    ,md_classes = md_classes m1 ++ md_classes m2
-                    }
+        g m1 m2 = ModuleDoc
+            { md_anchor = md_anchor m1
+            , md_name = md_name m1
+            , md_descr = md_descr m1
+            , md_adts = md_adts m1 ++ md_adts m2
+            , md_functions = md_functions m1 ++ md_functions m2
+            , md_templates = md_templates m2 ++ md_templates m2
+            , md_templateInstances =
```

Comment:
Is this an unrelated bugfix or did we not have to merge template instances before for some reason.

Reply:
Yes, it was a bug.